### PR TITLE
RCB-616: update python Jenkins script

### PR DESCRIPTION
**Pull request by yalcantara**
This PR is needed to update the reference for the python examples in Dockerhub

### CODE REVIEW

**Review by matt-l-01 — COMMENTED** [review]
Everything looks good from my end...I am going to get a second opinion on this before approval because this is my first verification and a larger verify!

File: .vscode/launch.json
Comment (seth-basistech) [inline .vscode/launch.json:1]: I think this file can be deleted.  You can probably add `.vscode` to `.gitignore` too.

File: CI.Jenkinsfile
Comment (seth-basistech) [inline CI.Jenkinsfile:23]: Leave a TODO comment somewhere in here to pull out the Sonar run to a single Python version (the latest?).  We only need to download the archive and execute it once.  Not for each Python version.

File: CI.Jenkinsfile
Comment (seth-basistech) [inline CI.Jenkinsfile:3]: Can you update the `classifiers` in `setup.py` to match this list.

**Review by seth-basistech — CHANGES_REQUESTED** [review]
This PR should include the removal of the deprecated code too so that it is ready to go in the next release cycle.

File: CI.Jenkinsfile
Comment (yalcantara) [reply CI.Jenkinsfile:25]: TODO comment added!

File: CI.Jenkinsfile
Comment (yalcantara) [reply CI.Jenkinsfile:3]: Updated!

**Pull request merged**
